assets/src/util/dataset.tsx

```diff
@@ -0,0 +1,23 @@
+// Functions to get values from the #app div's data attributes.
+
+/**
+ * Gets the entire dataset.
+ */
+export const getDataset = () => document.getElementById("app")?.dataset ?? {};
+
+/**
+ * Gets one value from the dataset, or undefined if the key is not present.
+ */
+export const getDatasetValue = (key: string) => getDataset()[key];
```

Comment:
Just curious, why do we need both `getDatasetValue` and `fetchDatasetValue`? I know one function throws and the other doesn't, but wouldn't we always want to throw?

Reply:
In some cases we're ok with the value not being set, e.g. [the Sentry DSN](https://github.com/mbta/screens/blob/master/lib/screens_web/templates/v2/screen/index.html.eex#L9-L11). `fetch` is for when it has to be there, `get` is for when we expect it to sometimes be undefined. I was going off of Elixir's `Map.get` and `Map.fetch!` functions.

However if you think it makes more sense, I could remove `getDatasetValue` and have the server always set _some_ value (maybe empty string or "none") for every expected data attribute.

Reply:
Ahh I see. Didn't think about those cases. I think two util functions is better than always setting the value in that case. I could imagine that adding new optional attributes could cause errors until we remember that _something_ needs to be there. So I like this.